### PR TITLE
Update the Quick Start Guide based on the new Statement Editor

### PR DESCRIPTION
**Pull request by RukshiW**
Fixes https://github.com/wso2-enterprise/choreo/issues/14912

### CODE REVIEW

File: en/docs/get-started/quick-start-guide.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ This quick start guide walks you through the steps to create, develop, publish,
 
 ### Step 1: Create a REST API
 
+First, let's create a new REST API:
+
 1. Sign in to the Choreo Console at [https://console.choreo.dev](https://console.choreo.dev).
 
 2. On the **Home** page, scroll to the **Use a sample and get started** section.
```
Comment (Nashaath) [inline en/docs/get-started/quick-start-guide.md:18]:
```suggestion
2. On the **Home** page, scroll to the **Get started with a template** section. 
```

File: en/docs/get-started/quick-start-guide.md [hunk 2]
```diff
@@ -91,25 +95,41 @@ Follow this procedure to further develop the API by adding a new resource and si
```
Comment (Nashaath) [inline en/docs/get-started/quick-start-guide.md:95]: We might need to update steps 3 and 4 to align with the new UIs. Might have to update the screenshot related to step 4 too.

Comment (RukshiW) [reply en/docs/get-started/quick-start-guide.md:95]: Done

File: en/docs/get-started/quick-start-guide.md [hunk 3]
```diff
@@ -68,7 +72,7 @@ Follow this procedure to further develop the API by adding a new resource and si
  
     ![Edit with VS Code Online](../assets/img/get-started/edit-with-vs-code.png){.cInlineImage-full}
 
-3. Click **+** on the low-code diagram, click **Resource**, and configure it as follows: 
+3. Click **+** on the low-code diagram to nopen the resource, and configure it as follows:
```
Comment (Nashaath) [inline en/docs/get-started/quick-start-guide.md:75]:
```suggestion
3. Click **+** on the low-code diagram to open the resource, and configure it as follows: 
```

File: en/docs/get-started/quick-start-guide.md [hunk 4]
```diff
@@ -79,7 +83,7 @@ Follow this procedure to further develop the API by adding a new resource and si
      | **Return Type** | `string|error` |
```
Comment (Nashaath) [inline en/docs/get-started/quick-start-guide.md:83]: Need to update some of the field names to align with the new names. For example, 
change **Path** to **Resource Path**
With regard to query parameter field names, change **Name** to **Param Name**

File: en/docs/get-started/quick-start-guide.md [hunk 5]
```diff
@@ -91,25 +95,41 @@ Follow this procedure to further develop the API by adding a new resource and si
 
  5. Click **Save**.
 
- 6. Click **Variable** and configure it as follows: 
+ 6. In the low-code diagram, click **+** after**Start**.
```
Comment (Nashaath) [inline en/docs/get-started/quick-start-guide.md:98]:
```suggestion
 6. In the low-code diagram, click **+** after **Start**.
```

File: en/docs/get-started/quick-start-guide.md [hunk 6]
```diff
@@ -91,25 +95,41 @@ Follow this procedure to further develop the API by adding a new resource and si
...
     ![Create Variable](../assets/img/get-started/create-variable.png){.cInlineImage-full}
 
-     | **Field** | **Value** |
-     |-----------------|-----------------------|
-     | **Variable Type** | `string` |
-     | **Variable Name** | `message` |
-     | **string** |`"Hello "+ name`|
+     In the **Add Constructs** pane, click **Variable**.
+
+ 7. Update the given variable statement syntax (i.e., **`var variable = <add-statement>`**) to generate a personalized message by following these steps:
```
Comment (Nashaath) [inline en/docs/get-started/quick-start-guide.md:104]:
```suggestion
 7. Update the given variable statement syntax (i.e., **`var variable = <add-expression>`**) to generate a personalized message by following these steps:
```

File: en/docs/get-started/quick-start-guide.md (hunk 4 above)
Comment (Nashaath) [reply en/docs/get-started/quick-start-guide.md:83]: Also might have to change the Return Type value to `string`. specifying as `string|error` results in an error later when adding the return expression.

Comment (RukshiW) [reply en/docs/get-started/quick-start-guide.md:83]: Updated the parameter name.

It works with `string|error` as the return type as agreed offline.

File: en/docs/get-started/quick-start-guide.md [hunk 7]
```diff
@@ -68,48 +72,68 @@ Follow this procedure to further develop the API by adding a new resource and si
...
+        string message = "Hello "+ name;
+        ```
 
- 7. Click **Save**.
+ 8. Click **Save**.
 
- 8. To return the personalized message, click **+** below the variable you created, click **Return**, and enter `message` as the **Return Expression**.
-    ![Add Return Type](../assets/img/get-started/add-return-type.png){.cInlineImage-full}
-    ![Select Return](../assets/img/get-started/select-return.png){.cInlineImage-full}
+ 9. To return the personalized message, click **+** below the variable you created, click **Return**.
```
Comment (Nashaath) [inline en/docs/get-started/quick-start-guide.md:126]:
```suggestion
 9. To return the personalized message, click **+** below the variable you created, and click **Return**. 
```

File: en/docs/get-started/quick-start-guide.md [hunk 8]
```diff
@@ -68,48 +72,68 @@ Follow this procedure to further develop the API by adding a new resource and si
...
+ 8. Click **Save**.
 
- 8. To return the personalized message, click **+** below the variable you created, click **Return**, and enter `message` as the **Return Expression**.
-    ![Add Return Type](../assets/img/get-started/add-return-type.png){.cInlineImage-full}
-    ![Select Return](../assets/img/get-started/select-return.png){.cInlineImage-full}
+ 9. To return the personalized message, click **+** below the variable you created, click **Return**. 
+
+     ![Add Return Type](../assets/img/get-started/add-return-type.png){.cInlineImage-full}
+ 
+     In the **Return** pane, the syntax for the return statement is dispalyed as **`return <add-expression>`**.
```
Comment (Nashaath) [inline en/docs/get-started/quick-start-guide.md:130]:
```suggestion
     In the **Return** pane, the syntax for the return statement is displayed as **`return <add-expression>`**.
```

File: en/docs/get-started/quick-start-guide.md [hunk 9]
```diff
@@ -68,48 +72,68 @@ Follow this procedure to further develop the API by adding a new resource and si
...
+
+     Click **`<add-expression`**, and click **message** in the list of suggestions that appear below the return statement syntax.
+
+      ![Select return expression](../assets/img/get-started/select-return-expression.png){.cInlineImage-half}
    
- 9. Click **Save**.
+ 10. Click **Save**.
 
 !!!info
     You can click the **</>** icon on the top left corner of the editor if you want to work in the code view. Depending on your expertise, you can work in the low-code view or the pro-code view. You can also work in both views simultaneously.
```
Comment (Nashaath) [inline en/docs/get-started/quick-start-guide.md:139]:
```suggestion
    You can click the **</>** icon on the top left corner of the editor if you want to work in the code view. Depending on your expertise, you can work in the low-code view or the pro-code view. You can also work on both views simultaneously.    
```

**Review by Nashaath — APPROVED** [review]

**Pull request merged**